app/assets/javascripts/tour.js

```diff
@@ -0,0 +1,92 @@
+var Tether = require('tether');
+var Shepherd = require('tether-shepherd');
+var axios = require('axios');
+
+function getCurrentUser() {
+  return axios.get('/users/current');
+}
+
+function updateTour(user) {
+  var remainingSteps = user.tour_steps.filter(step => step.done !== true);
+
+  if(remainingSteps.length === 0)
+    user.tour = false;
+
+  var data = {
+    user: {
+      tour: user.tour,
+      tour_steps: JSON.stringify(user.tour_steps)
+    }
+  };
+
+  return axios.put(`/users/${user.id}/tour`, data);
```

Comment:
Those requests should be on a separate file, maybe on a `Tour` Backbone model to keep it organized, what do you think?